project.clj

```diff
@@ -3,4 +3,4 @@
   :url "https://github.com/weavejester/comb"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.2.1"]])
+  :dependencies [[org.clojure/clojure "1.8.0"]])
```

Comment:
The Clojure version in the dependencies typically denotes the minimum Clojure version we want the library to be compatible with. This is because it guarantees we won't accidentally use a feature from later versions of Clojure that makes the library compatible with earlier versions of Clojure.

I'd be fine with updating the minimum Clojure version to 1.5.1, but not to 1.8.0. I don't think there's any feature in 1.8.0 that was absolutely require.